.github/PULL_REQUEST_TEMPLATE/prometheusrules.md

```diff
@@ -0,0 +1,10 @@
+# Prometheus Rule change
+- [ ] This Pull Request fixes an open issue, Bugzilla or Jira Card
+- [ ] This Pull Request contains a reference to the issue, Bugzilla or Jira card it fixes
+- [ ] I have evaluate my rule according to the matrix below.
+
+| Criteria                    | Yes/No |
+|-----------------------------|--------|
+| works on vanilla Kubernetes |        |
+| works on vanilla OCP        |        |
+| works on managed Openshift  |        |
```

Comment:
Does ARO fall under this umbrella/criteria?

Reply:
Answered this one via mail